db/migrate/20150709221755_create_logins.rb

```diff
@@ -1,4 +1,4 @@
-class CreateLogins < ActiveRecord::Migration
+class CreateLogins < ActiveRecord::Migration[4.2]
```

Comment:
I think we need a version check as otherwise we'd have to remove support for old Rails versions.